plugins/workspace-backpack/src/options.js

```diff
@@ -44,6 +45,7 @@ export function parseOptions(options) {
       copyAllToBackpack: false,
       pasteAllToBackpack: false,
       disablePreconditionChecks: false,
+      allowEmptyBackpackOpen: true,
```

Comment:
This should be a new top-level option instead of being nested under contextMenu options. The context menu options configures which actions are available in the context menu, and this option is unrelated.